test/backup_test.js

```diff
@@ -266,14 +265,16 @@ describe('Backup', () => {
         return _.omit(model, ['id']);
       }
 
+      // We want to know which paths have two slashes, since that tells us which files
+      //   in the attachment fan-out are files vs. directories.
+      const TWO_SLASHES = /[^/]*\/[^/]*\/[^/]*/;
```

Comment:
Thanks for the comment. This makes it much clearer. Maybe we could create a function `const isDirectory = fileOrDir => TWO_SLASHES.test(fileOrDir)`.

Minor: With `String::match`, you can use a simple `/\//` pattern to count occurrences of a given string without having to create complicated patterns: https://stackoverflow.com/a/4009768